eng/Versions.props

```diff
@@ -82,8 +82,6 @@
     <!-- We are temporarily pinning sources packages and M.AspNetCore.Testing until we migrate this repo to aspnetcore -->
     <MicrosoftAspNetCoreTestingPackageVersion>5.0.0-preview.2.20121.3</MicrosoftAspNetCoreTestingPackageVersion>
     <MicrosoftAspNetCoreBenchmarkRunnerSourcesPackageVersion>5.0.0-preview.2.20121.3</MicrosoftAspNetCoreBenchmarkRunnerSourcesPackageVersion>
-    <MicrosoftExtensionsCommandLineUtilsSourcesPackageVersion>5.0.0-preview.2.20121.3</MicrosoftExtensionsCommandLineUtilsSourcesPackageVersion>
-    <MicrosoftExtensionsHashCodeCombinerSourcesPackageVersion>5.0.0-preview.2.20121.3</MicrosoftExtensionsHashCodeCombinerSourcesPackageVersion>
     <MicrosoftExtensionsNonCapturingTimerSourcesPackageVersion>5.0.0-preview.2.20121.3</MicrosoftExtensionsNonCapturingTimerSourcesPackageVersion>
     <BenchmarkDotNetPackageVersion>0.10.13</BenchmarkDotNetPackageVersion>
     <MicrosoftBuildFrameworkPackageVersion>15.8.166</MicrosoftBuildFrameworkPackageVersion>
```

Comment:
Later on in this file I believe you can remove all of the `Runtime_` versions now.

Reply:
Runtime_MicrosoftCodeAnalysisCSharpPackageVersion is still used by Razor.Test.Common which is used by CA.Razor.Workspaces.Test.Common

Reply:
oh interesting, that's probably a bug and should be replaced with the tooling variant.

Reply:
I'll try switching it.